server/src/internalClusterTest/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -1795,4 +1798,17 @@ public void testReservesBytesDuringPeerRecoveryPhaseOne() throws Exception {
             .mapToLong(n -> n.getIndices().getStore().getReservedSize().getBytes()).sum(), equalTo(0L));
     }
 
+    private void assertGlobalCheckpointIsStableAndSyncedInAllNodes(String indexName, int shardId) throws Exception {
+        long maxSeqNo = NO_OPS_PERFORMED;
```

Comment:
Maybe start the search with something obvious invalid like `Long.MIN_VALUE`?

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/75898/commits/20585d2aacbe324649fa273d039fc309d5ebc816